use-case-monorepo-project.md

```diff
@@ -0,0 +1,272 @@
+This guide shows you how to optimize your Semaphore workflow for monorepo
+projects.
```

Comment:
A gentle two-three paragraph introduction into "what is a monorepo" would be a great start for this guide.

Followed up by, "Semaphore comes out-of-box with support for monorepos".